### PR TITLE
Updated fr tn pe al

### PR DESCRIPTION
**Pull request by dct-infra**

### CODE REVIEW

File: channels/al.m3u [hunk 1]
```diff
@@ -11,6 +11,8 @@ http://us.bestvideostreaming.is/8737/playlist.m3u8
 https://shkoder.gjirafa.com/api/media/rgjirafa/t0110y/index.m3u8
 #EXTINF:-1 tvg-id="Cufo.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/cjVAXnZ.png" group-title="Kids",Çufo (576p)
 http://93.157.62.180/Cufo/index.m3u8
+#EXTINF:-1 tvg-id="EuronewsAlbania.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/hTBah1z.jpg" group-title="News",Euronews Albania (1080p)
```
Comment (freearhey) [inline channels/al.m3u:14]: Euronews Albania is a French channel: https://www.lyngsat.com/tvchannels/fr/EuroNews-Albania.html

**Review by freearhey — CHANGES_REQUESTED** [review]
The point is that we sort channels by country of broadcast source. To specify the target country you can use the `tvg-country` attribute.

File: channels/fr.m3u [hunk 2]
```diff
@@ -57,8 +57,6 @@ https://cdn02.hta.dz/abr_htatv/EL_HEDDAF_TV/htatv/EL_HEDDAF_TV_720p/chunks.m3u8
 https://emci-fr-hls.akamaized.net/hls/live/2007265/emcifrhls/04.m3u8
 #EXTINF:-1 tvg-id="EMCITVMontreal.fr" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",EMCI TV Montreal (1080p)
 https://emci-td-hls.akamaized.net/hls/live/2007264/emcitdhls/index.m3u8
-#EXTINF:-1 tvg-id="EuronewsAlbania.fr" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/hTBah1z.jpg" group-title="News",Euronews Albania [Offline]
-http://51.195.88.12:4050/live/euronewsmobpp2/playlist.m3u8
```
Comment (Dum4G) [inline channels/fr.m3u:61]: https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UChR-A__NS_C5kHDWj3PeAhw/live

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/al.m3u [hunk 3]
```diff
@@ -11,6 +11,8 @@ http://us.bestvideostreaming.is/8737/playlist.m3u8
...
+https://iptv.dc-team.com/query?url=https://www.youtube.com/watch?v=Sw2XgYV8GbI
```
Comment (Dum4G) [inline channels/al.m3u:15]: That link will expire eventually. Use channel link + "/live"

File: channels/fr.m3u (hunk 2 above)
Comment (dct-infra) [reply channels/fr.m3u:61]: Thanks for the link, couldn't find that one.

File: channels/al.m3u (hunk 1 above)
Comment (dct-infra) [reply channels/al.m3u]: I get it now, I thought the listing was based on the language

**Review by freearhey — APPROVED** [review]

File: channels/al.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/al.m3u]: They stream 2 things and they keep the most recent stream to show on live. So it isn't easy...

**Review by Dum4G — APPROVED** [review]

**Pull request merged**